ansible_deploy.sh

```diff
@@ -20,12 +20,29 @@ else
   brew install ansible
 fi
 
-# Run Ansible playbook
-if ( curl -I pi.attlocal.net:80 )
+# Dynamically add IP to inventory.ini file if Pi is running
+if ( dig +short pi.attlocal.net )
```

Comment:
Maybe make the hardcoded `pi.attlocal.net` an environment variable that is fed to the script when you run it like `export PI_FQDN=pi.attlocal.net; bash ansible_deploy.sh`